webdriver/tests/support/fixtures_bidi.py

```diff
@@ -101,21 +103,38 @@ async def on_event(_, data):
         remove_listener()
 
 
+@pytest.fixture
+def wait_for_future_safe(configuration):
```

Comment:
I assume that we can also use this new fixture for the two usages of `asyncio.shield` under `browsing_context` as well? If yes, feel free to replace so that we don't forget about it.

Reply:
I think so too. I am planning to send another PR that uses this fixture in all places where it's possible, starting with the network module.